ext/sodium/config.m4

```diff
@@ -1,5 +1,5 @@
 dnl $Id$
-dnl config.m4 for extension libsodium
+dnl config.m4 for extension sodium
 
 PHP_ARG_WITH(libsodium, for libsodium support,
 [  --with-libsodium[[=DIR]]  Include libsodium support])
```

Comment:
I don't think there should be `--enable-sodium` as it depends on the library. From what I know, enable is just for extensions without lib dependency and for lib dependency we always have `--with` or am I missing anything?